### ISSUE TITLE
Coloring of segmentation images

### ISSUE THREAD

**Issue by tischi** [embl-cba/fiji-plugin-cats#21]
@davodogster

The label mask class images that can be outputted now can be coloured running this IJ macro command:

`run("glasbey_inverted");`

Or in the Menu:

<img width="336" alt="image" src="https://user-images.githubusercontent.com/2157566/59900000-1335aa00-93f7-11e9-8a9b-574e97c82c93.png">

The applies the `Glasbey LUT` to the image which gives each gray value a different color.

In below example the background class has a label of 1, and the class for the blobs has a label of 2.

<img width="742" alt="image" src="https://user-images.githubusercontent.com/2157566/59899948-dff31b00-93f6-11e9-9af9-cdb0f925d53e.png">

Shall I automatically apply this LUT to the segmentation images? I think it would be a good idea...

**Comment by davodogster** [comment]
Yes please I think it should automatically be applied and work for a single image or a folder of images.


________________________________

**Comment by tischi** [comment]
Ok. I will do this.

**Comment by tischi** [comment]
Add class colouring to segmentation output.

**Comment by davodogster** [comment]
Hi Christian can you enable us to apply a classifier to a folder of images and save all of the classification outputs with class coloring multiple labels in one image.

**Comment by davodogster** [comment]
Also, I just used the advanced classification option (can only classify one image, not a directory), and I selected to output all probabilities in one image (binary classification). The result was a multiclass grayscale with many different values representing many different probabilities.
